conda-store-server/conda_store_server/api.py

```diff
@@ -322,6 +322,20 @@ def get_build_packages(
     )
 
 
+def get_build_lockfile_legacy(db, build_id: int):
+    build = db.query(orm.Build).filter(orm.Build.id == build_id).first()
+    packages = [
+        f"{row.package.channel.name}/{row.subdir}/{row.package.name}-{row.package.version}-{row.build}.tar.bz2#{row.md5}"
+        for row in build.package_builds
+    ]
+    return """#platform: {0}
+@EXPLICIT
+{1}
+""".format(
+        conda_utils.conda_platform(), "\n".join(packages)
+    )
```

Comment:
Same as https://github.com/conda-incubator/conda-store/blob/v0.4.15/conda-store-server/conda_store_server/api.py#L219 (with minor changes: function name and qualified import of `conda_platform`):

```py
def get_build_lockfile(db, build_id: int):
    build = db.query(orm.Build).filter(orm.Build.id == build_id).first()
    packages = [
        f"{row.package.channel.name}/{row.subdir}/{row.package.name}-{row.package.version}-{row.build}.tar.bz2#{row.md5}"
        for row in build.package_builds
    ]
    return """#platform: {0}
@EXPLICIT
{1}
""".format(
        conda_platform(), "\n".join(packages)
    )
```

Reply:
Have always preferred qualified imports :smile: